pages/common/xml-elements.md

```diff
@@ -0,0 +1,28 @@
+# xml-elements
+
+> Extract elements and structure of an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Extract elements of an XML document (as XPATH expressions):
+
+`xml elements {{path/to/input.xml|URI}} >{{path/to/elements}}`
```

Comment:
I don't see an XPATH expression here? Or does it generate XPATH expressions as output? I'm confused.

Reply:
- Extract elements from an XML document (producing XPATH expressions):